docker-compose.production.yml

```
@@ -8,10 +8,10 @@ version: "3"
 services:
   web:
     restart: always
-    hostname: "$HOSTNAME"
+    hostname: ${HOSTNAME:-$HOST}
     environment:
       - GUNICORN_OPTS= --workers 50 --timeout 300 --max-requests 500
       - OL_CONFIG=/olsystem/etc/openlibrary.yml
-      - PYENV_VERSION=3.8.6
+      - PYENV_VERSION=${PYENV_VERSION:-"3.8.6"}
```

This forces it to be 3.8.6; i.e. you don't need to set it from the command line. `${PYENV_VERSION:-"3.8.6"}` has a typo -- should be `${PYENV_VERSION:-3.8.6}`

```suggestion
      - PYENV_VERSION=3.8.6
```